community-operators/federation/federation.v0.0.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,189 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: federation.v0.0.5
+  namespace: placeholder
+  annotations:
+    categories: "OpenShift Optional, Integration & Delivery"
+    description: "Kubernetes Federation namespace-scoped installation"
+spec:
+  displayName: Federation
+  description: |
+    This deploys Federation in a namespace-scoped configuration. After
+    installation, you must enable the API types you want Federation to manage
+    using `kubefed2 enable`.
+
+    Note, you must enable federation support for namespaces in this
+    configuration before other types will work. Use `kubefed2 enable namespaces`.
+  version: 0.0.5
+  maturity: alpha
+  provider:
+    name: Red Hat
+  labels:
+    alm-owner-federation: federation
+    alm-status-descriptors: federation.v0.0.5
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: false
```

Comment:
Is the intent that this operator only watch the namespace it's deployed in?

Reply:
Following the documentation on InstallModes [here](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md#operator-metadata), I believe that SingleNamespace should also be set to true. Having both OwnNamespace and SingleNamespace would mean that the operator can only be installed into an OperatorGroup with one watched namespace, and that namespace must be the same namespace as operator is deployed to. OLM cannot install the operator otherwise.

Reply:
fixed, ptal